src/rulesets/LinearAlgebra/dense.jl

```diff
@@ -163,6 +163,25 @@ function rrule(::typeof(*), B::AbstractArray{<:Real}, A::Real)
     return A * B, times_pullback
 end
 
+#####
+##### `\`, `/` matrix-scalar_rule
+
+function rrule(::typeof(/), A::AbstractArray{<:Real}, b::Real)
```

Comment:
Are these maybe a bit too abstractly typed?

Reply:
I don' t think so in this case.
But I also don't think this is the PR to be having the discussion about it in.
This matchs to `*` defined above.
I think changing `*` and changing this can happen at same time

Reply:
Fair enough.